eventually/authentication/models.py

```diff
@@ -34,7 +35,22 @@ class CustomUser(AbstractBaseUser):
 
     def __str__(self):
         """
-            This magic method is redefined to show first, last name and id of the CustomUser object.
+        This magic method is redefined to show first, last name and id of the CustomUser object.
+        :return: Full string representation of the object
+        """
+        return '{} {} {} {} {} {} {} {}'.format(self.id,
```

Comment:
can you find a better approach to make the same result but in a more readable way?

Reply:
Could you specify what should be more readable: the code or the output result?
Guess I have to add the names of the fields to the ouput?